test/src/test_register.rs

```diff
@@ -232,3 +235,61 @@ fn test_advanced_methods() -> bool {
 
     ok
 }
+
+#[derive(NativeClass)]
+#[inherit(Reference)]
+#[register_with(VarargsGets::register)]
+struct VarargsGets {}
+
+#[methods]
+impl VarargsGets {
+    fn new(_owner: TRef<Reference>) -> Self {
+        Self {}
+    }
+
+    fn register(builder: &ClassBuilder<VarargsGets>) {
+        builder.method("calc", CalcMethod).done();
+    }
+}
+
+struct CalcMethod;
+
+impl Method<VarargsGets> for CalcMethod {
+    fn call(
+        &self,
+        _this: TInstance<'_, VarargsGets>,
+        args: gdnative::export::Varargs<'_>,
+    ) -> Variant {
+        (|| {
+            args.check_length(1..=3)?;
+            let a: i64 = args.get(0)?;
+            let b: i64 = args.get(1)?;
+            let c: i64 = args.get_opt(2)?.unwrap_or(11);
+
+            let ret = a * b - c;
+            Ok::<Variant, Box<dyn Error>>(ret.to_variant())
+        })()
+        .unwrap_or_default()
```

Comment:
Should we just return the default value if there is a type error occurring?
Or maybe rather `unwrap()` which causes a panic (or `expect()` with a message).

Reply:
There, I needed to use `godot_error!()` to output the error. I will fix it!